Jenkinsfile

```diff
@@ -39,11 +40,6 @@ pipeline {
 
         // Docker build flags are set via the getDockerBuildFlags() shared library.
         sh "docker build ${getDockerBuildFlags()} -t rightbrainnetworks/auto-semver:${env.VERSION} ."
```

Comment:
We can do this with native jenkins docker.build or something.

Reply:
I'm unsure

Reply:
https://jenkins.io/doc/book/pipeline/docker/#building-containers